code-challenges.test.js

```diff
@@ -56,6 +143,28 @@ var hand2 = [5, 5, 3, 3, 4]
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
-
-
 // b) Create the function that makes the test pass.
+
+//Create a function called isFullHouse
+//  parameters - array of 5 numbers
+//  create an array of 13 0's to represent the 13 cards count
+//  iterate over the hand and whatever the value of the card is add 1 to the count for that value in the count array
+//  A full house should have 3 and 2 of the same values so check to make there is a count of 3 and a count of 2
+//  if true return true else return false
+//END
+
+const isFullHouse = (hand) => {
+    handCountArr = Array(13).fill(0);
+   
+    for(let i = 0; i < hand.length; i++){
+        handCountArr[hand[i]-1] += 1 
+    }
+    if(handCountArr.includes(3) && handCountArr.includes(2)){
+        return true
+    }
+    else{
+        return false
+    }
+}
```

Comment:
Creative! I appreciate you incorporated all the values of the cards including royals and ace